pkgs/development/libraries/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib, stdenv, fetchFromGitLab,
```

Comment:
Oh right, I believe the nixpkgs policy is to write the parameters one per line. Helps keep diffs a bit cleaner when adding / removing stuff in the future.

Reply:
Seems to be all over the place, but it makes sense. Added some newlines.